api/v1alpha1/release_types.go

```diff
@@ -23,20 +23,20 @@ import (
 	"time"
 )
 
-// ReleaseSpec defines the desired state of Release
+// ReleaseSpec defines the desired state of Release.
 type ReleaseSpec struct {
 	// ApplicationSnapshot to be released
 	// +kubebuilder:validation:Pattern=^[a-z0-9]([-a-z0-9]*[a-z0-9])?$
 	// +required
 	ApplicationSnapshot string `json:"applicationSnapshot"`
 
-	// ReleaseLink referencing the workspace where the snapshot will be released
+	// ReleasePlan to use for this particular Release
```

Comment:
Maybe _ReleasePlan used_ instead or _to use_? Not sure the correct grammar here, but this sentence sounded confusing to me.

Reply:
Nothing used when you define a Release. It get's used later on. I think it's ok.